cardano-cli/test/Test/Cli/ITN.hs

```diff
@@ -1,6 +1,6 @@
 {-# LANGUAGE OverloadedStrings #-}
 
-module Test.ITN
+module Test.Cli.ITN
```

Comment:
Should we stick with `Test.CLI.*` for better consistency with the rest of the project? i.e. `Cardano.CLI.*`

Although, I actually do prefer `*.Cli.*` overall.

Reply:
I think we should change the others to be like this :grinning: .

Reply:
I also happen to prefer `Cli` 🤷